packages/web-components/fast-components/src/breadcrumb/breadcrumbs.stories.ts

```diff
@@ -0,0 +1,15 @@
+import { FASTDesignSystemProvider } from "../design-system-provider";
```

Comment:
Should this file be named `breadcrumb.stories.ts` or `breadcrumbs.stories.ts`? Same for the others which are inconsistent.

Reply:
I think breadcrumbs.

Reply:
Let’s make sure all files in the folder are named consistently.

Reply:
FWIW, the ARIA spec refers to “Breadcrumb”.

Reply:
We should strive to align in the direction that Open-UI is most likely to go. I’ll reach out to a couple folks.

Reply:
7/9 on OpenUI seem to be named "Breadcrumb" are there others which are named "Breacrumbs" besides the two outliers?

Reply:
And to be clear - talking about the file names; the control appears to be named "Breadcrumb"

Reply:
I have only seen those two